pkg/virt-operator/resource/generate/rbac/controller.go

```diff
@@ -227,6 +227,17 @@ func newControllerClusterRole() *rbacv1.ClusterRole {
 					"update",
 				},
 			},
+			{
+				APIGroups: []string{
+					"",
+				},
+				Resources: []string{
+					"pods/status",
+				},
+				Verbs: []string{
+					"update",
```

Comment:
Who needs access to the pod/status for updates? It smells fishy to me.

Reply:
This is templating the virt-controller pod's RBAC.

I am adding it the ability of being able to update the pod/status - which includes the pod's metadata, where the annotation's are located - those are what I need to update.

You're right that if I patch - or if I do `Update` instead of `UpdateStatus` I wouldn't need this, since the controller already has RBAC permissions for that: https://github.com/kubevirt/kubevirt/blob/d3971acae8feb03c7d209154a0567df9e89abbf1/pkg/virt-operator/resource/generate/rbac/controller.go#L194

Let's revisit this once I patch the annotation instead of updating the pod.

Reply:
I still do not understand how `pod/status` has anything to do with the `metadata` fields.

Reply:
That's a good point. This code is 1 year old already and I can't recollect why I did it in the first place. That's sad.

I'm (for the time being ...) removing this and run a regular `Update` - this allows me to remove the RBAC changes from the PR.

In the coming days I'll check your suggestion of using a patch instead of an update.

Reply:
Resolving since I no longer use `updateStatus`.